change/@office-iss-react-native-win32-2020-03-27-15-10-36-no-tag-60.json

```diff
@@ -0,0 +1,8 @@
+{
+  "comment": "Do Not Override NPM Tag Based on Branch Name (#4434)",
```

Comment:
Should this have a "type": "none"?  -- Not sure what happens if the type field is missing.

Reply:
That's bizarre... This commit was generated by Beachball, so I don't know what happened here. @kenotron any idea why this would happen?